contracts/sfc/ConstantsManager.sol

```diff
@@ -87,34 +83,14 @@ contract ConstantsManager is Ownable {
         treasuryFeeShare = v;
     }
 
-    function updateUnlockedRewardRatio(uint256 v) external virtual onlyOwner {
+    function updateRewardRatio(uint256 v) external virtual onlyOwner {
```

Comment:
I recommend to not rename this method, if we suppose the locking will be added back later.